projects/metering/setup-metering-project.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
We haven't really nailed down a rigid approach but generally we:

 - want RBAC and cluster-scoped resources that require elevated privilges in separate targets
 - want YAML over shell
 - want make to be doing the work in composable targets

Reply:
Also colocating e.g. ci-operator RBAC ... that being said, i see the appeal of having your own self-managed thing that DPTP does not care to look at. So this could be fine too. @smarterclayton any strong thoughts?

Reply:
> want make to be doing the work in composable targets

I found that currently the way make is being used is very prone to misuse due to the fact that it relies heavily on phony targets and any use of `-j` would break most of the way things assume it will work today, so I opted to use shell instead.

I can update things to be more independent, but I believe using Make for that can be prone to misordering if incorrectly called, but I can update things to fall-in-line if that's what you want.

Reply:
@stevekuznetsov @smarterclayton How should this proceed?

Reply:
We can keep this the wild west until DPTP needs to manage any of it

/lgtm
/approve
/hold cancel
/woof

Reply:
Sounds good to me. Makes your life easier!